src/pages/ConditionsList/index.tsx

```diff
@@ -107,17 +108,21 @@ export const ConditionsList: React.FC = () => {
 
   // Clear the filters
   React.useEffect(() => {
-    setSelectedOracleValue(OracleFilterOptions.All)
-    setSelectedOracleFilter([])
-    setSelectedConditionTypeValue(ConditionTypeAll.all)
-    setSelectedConditionTypeFilter(null)
-    setSelectedStatus(StatusOptions.All)
-    setSelectedMinOutcomes(null)
-    setSelectedMaxOutcomes(null)
-    setSelectedToCreationDate(null)
-    setSelectedFromCreationDate(null)
-    setSearchBy(ConditionSearchOptions.ConditionId)
-    setTextToSearch('')
+    if (!showFilters) {
+      setResetPagination(!resetPagination)
+      setSelectedOracleValue(OracleFilterOptions.All)
+      setSelectedOracleFilter([])
+      setSelectedConditionTypeValue(ConditionTypeAll.all)
+      setSelectedConditionTypeFilter(null)
+      setSelectedStatus(StatusOptions.All)
+      setSelectedMinOutcomes(null)
+      setSelectedMaxOutcomes(null)
+      setSelectedToCreationDate(null)
+      setSelectedFromCreationDate(null)
+      setSearchBy(ConditionSearchOptions.ConditionId)
+      setTextToSearch('')
+    }
```

Comment:
Just for clarity, this could be a resetFilters function?

Reply:
Is used only in one place, I think we don't need a function